amundsen_application/static/js/components/ProfilePage/index.spec.tsx

```diff
@@ -366,6 +349,17 @@ describe('ProfilePage', () => {
       );
     });
 
+    it('renders alumni bullet is user not active', () => {
+      const userCopy = {
+        ...globalState.user.profile.user,
+        is_active: false,
+      };
+      const { wrapper } = setup({
+        user: userCopy,
+      });
+      expect(wrapper.find('#alumni').text()).toEqual('Alumni');
```

Comment:
I would avoid to check for text on the UI, this makes this test fragile

Reply:
Could also check against a constant value.